tracing-subscriber/src/registry/mod.rs

```diff
@@ -1,6 +1,67 @@
 //! Storage for span data shared by multiple [`Layer`]s.
 //!
-//! [`Layer`]: ../layer/struct.Layer.html
+//! ## Using the Span Registry
+//!
+//! This module provides the [`Registry`] type, a [`Subscriber`] implementation
+//! which tracks per-span data and exposes it to [`Layer`]s. When a `Registry`
+//! is used as the base `Subscriber` of a `Layer` stack, the
```

Comment:
Is it not the case that `Registry` is _always_ used as the base `Subscriber`?

Reply:
I meant this as "If a stack uses a `Registry` as the base subscriber...", not "if a registry, which can be used in multiple ways, is used as the base...". Will reword it!

Reply:
Not necessarily—other registries can be provided and introduced. I'll call that out.